packages/lit-dev-content/site/blog/2022-10-18-lit-labs-event.md

```diff
@@ -0,0 +1,38 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Get the low-down on Lit Labs"
+summary: "Join us for a virtual event on October 24th 2022 at 10AM PDT to learn all about Lit Labs!"
+date: 2022-10-18
+author:
+  - elliott-marquez
+eventDate: "Monday, October 24th 2022 at 10AM PDT"
+ytLink: https://www.youtube.com/watch?v=l6Gn5uV83sw
+---
+
+The Lit team is hosting a virtual event on **{{eventDate}}** to talk about all of the exciting work going on in Lit Labs. We'll give an overview of the Lit Labs program, followed by detailed presentations on several Labs packages. And we'll let you know how you can help get these packages across the finish line and shape the future of Lit!
+
+To make sure you don't miss it, [**go to the live stream**]({{ytLink}}) and sign up for notifications.
+
+<img src="/images/blog/labs-event/lit-labs.svg" width="720" class="block centered">
+
+## What is Lit Labs?
+
+[Lit Labs]({{site.baseurl}}/docs/libraries/labs/) is an umbrella for Lit packages under development that we are actively seeking feedback on. These projects may be experimental or incomplete, or simply looking for real-world feedback. Lit Labs projects are published under the [`@lit-labs` npm scope](https://www.npmjs.com/search?q=%40lit-labs).
```

Comment:
```suggestion
[Lit Labs]({{site.baseurl}}/docs/libraries/labs/) is an umbrella for Lit packages under development that we are actively seeking feedback on. These projects may be experimental or incomplete, or simply looking for real-world testing. Lit Labs projects are published under the [`@lit-labs` npm scope](https://www.npmjs.com/search?q=%40lit-labs).
```

If you don't want to repeat the word "feedback," you could go with "real-world testing" perhaps?

Reply:
Hmm not sure if it's the same. I think we already have a lot of testing happening, but we don't necessarily know how it's going